tests/Proto.Remote.Tests/RemoteTests.cs

```diff
@@ -38,9 +38,25 @@ public class RemoteTests
             });
         }
 
+        [Fact]
+        public async void CanSpawnRemoteActor()
+        {
+            var props = Actor.FromProducer(() => new LocalActor());
+            var pid = Actor.Spawn(props);
+            var remoteActorName = Guid.NewGuid().ToString();
+            var remoteActor = await Remote.SpawnNamedAsync("127.0.0.1:12000", remoteActorName, "remote", TimeSpan.FromSeconds(5));
+            await remoteActor.RequestAsync<Start>(new StartRemote { Sender = pid }, TimeSpan.FromMilliseconds(2500));
+            remoteActor.Tell(new Ping());
+            await Task.Delay(TimeSpan.FromSeconds(2));
```

Comment:
Agree, making it use ReceiveAsync would be better and will fail less often on the build server